spec/features/all_pages_spec.rb

```diff
@@ -17,6 +17,11 @@
   end
 
   context 'Event show' do
+    it 'Shows name in title' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.